src/index.html

```diff
@@ -22,20 +22,32 @@ <h3>What is Cassandra?</h3>
-        <a href="http://planetcassandra.org/blog/post/godaddy-worlds-largest-domain-name-registrar-and-web-host-provider-utilizes-cassandra-for-replication-and-scalability/">GoDaddy</a>,
-        <a href="http://planetcassandra.org/blog/post/cassandra-used-to-build-scalable-and-highly-available-systems-at-hulu-streaming-content-to-over-5-million-subscribers/">Hulu</a>,
-        <a href="http://planetcassandra.org/blog/post/instagram-making-the-switch-to-cassandra-from-redis-75-instasavings/">Instagram</a>, 
-        <a href="http://www.slideshare.net/planetcassandra/3-mohit-anchlia">Intuit</a>,
-        <a href="http://www.slideshare.net/adrianco/migrating-netflix-from-oracle-to-global-cassandra">Netflix</a>,
-        <a href="http://planetcassandra.org/blog/post/reddit-upvotes-apache-cassandras-horizontal-scaling-managing-17000000-votes-daily/">Reddit</a>, 
-        <a href="http://planetcassandra.org/blog/post/make-it-rain-apache-cassandra-at-the-weather-channel-for-severe-weather-alerts/">The Weather Channel</a>,
-        and <a href="http://planetcassandra.org/companies/">over 1500 more companies</a> that have large, active data
-        sets.</p>
+
+        <!-- <a href="http://planetcassandra.org/blog/post/analytics-at-github-with-apache-cassandra/">GitHub</a>, -->
+        <!-- <a href="http://planetcassandra.org/blog/post/godaddy-worlds-largest-domain-name-registrar-and-web-host-provider-utilizes-cassandra-for-replication-and-scalability/">GoDaddy</a>, -->
+        <!-- <a href="http://planetcassandra.org/blog/post/cassandra-used-to-build-scalable-and-highly-available-systems-at-hulu-streaming-content-to-over-5-million-subscribers/">Hulu</a>, -->
+        <!-- <a href="http://planetcassandra.org/blog/post/instagram-making-the-switch-to-cassandra-from-redis-75-instasavings/">Instagram</a>,--> 
+
+	<a href="https://www.datastax.com/blog/2020/04/accelerate-rewind-how-instagram-uses-cassandra-operate-global-scale-2">Instagram</a>,
+
+        <!-- <a href="http://www.slideshare.net/planetcassandra/3-mohit-anchlia">Intuit</a>,-->
+	<a href="https://www.datastax.com/resources/video/datastax-accelerate-2019-managing-400-billion-requests-zero-downtime-turbotax">Intuit</a>, 
+
+        <!-- <a href="http://www.slideshare.net/adrianco/migrating-netflix-from-oracle-to-global-cassandra">Netflix</a>, -->
+	<a href="https://www.slideshare.net/VinayKumarChella/how-netflix-manages-petabyte-scale-apache-cassandra-in-the-cloud">Netflix</a>, 
+
+        <!-- <a href="http://planetcassandra.org/blog/post/reddit-upvotes-apache-cassandras-horizontal-scaling-managing-17000000-votes-daily/">Reddit</a>, -->
+        <!-- <a href="http://planetcassandra.org/blog/post/make-it-rain-apache-cassandra-at-the-weather-channel-for-severe-weather-alerts/">The Weather Channel</a>,-->
+        and  thousands of other companies
+<!-- <a href="http://planetcassandra.org/companies/">over 1500 more companies</a> -->
```

Comment:
aside: it's a shame there's no longer an equivalent to this link.

Reply:
Yes, it is. I did do some digging, but the net result is that there is no such list freely available.